test/long-haul/cleanup/cleanupjob.yaml

```diff
@@ -7,6 +7,8 @@ provisioner: azurelustre.csi.azure.com
 parameters:
   mgs-ip-address: "{lustre_fs_ip}"
   fs-name: "{lustre_fs_name}"
+  sub-dir: "longhaul"
```

Comment:
Is this a concern for a shared cluster with testing? Uniquify?

Reply:
I know we've had setting up a cni longhaul pipeline on the backburner so maybe uniquify so we can set that up and point to the same cluster?

But we can also set up a 2nd cluster so leaving it as-is is fine too

Reply:
The existing logic runs this task to rm -rf everything on the volume, so if it anything this is now less destructive. We can revisit the general approach when extending this testing